releases/v1.1-beta.20170907.md

```diff
@@ -0,0 +1,155 @@
+---
+title: What&#39;s New in v1.1-beta.20170907
+toc: false
+summary: Additions and changes in CockroachDB version v1.1-beta.20170907
+---
+
+## Aug 31, 2017
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-beta.20170907.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-beta.20170907.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-beta.20170907.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1-beta.20170907.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- `DROP DATABASE` no longer drops non-empty databases unless the `CASCADE` modifier is added. [#17926](https://github.com/cockroachdb/cockroach/pull/17926)
+- The `--cache` and `--max-sql-memory` flags now default to 128MiB instead of 25% of physical memory to avoid using too much memory by default. Production deployments are encouraged to set `--cache` explicitly to 25-50% of the memory allocated to CockroachDB for best performance. [#18040](https://github.com/cockroachdb/cockroach/pull/18040)
+
+
+### SQL Language Changes
+
+- The new [`IMPORT TABLE`](../v1.1/import-table.html) statement provides a fast way to import an entire table's data into CockroachDB by loading CSV files in a distributed fashion. Although this feature uses [CCL-licensed code](https://www.cockroachlabs.com/cockroachdb-community-license/), and is therefore available only in builds that include CCL-licensed code, using it is free and does not require an enterprise license. [#17539](https://github.com/cockroachdb/cockroach/pull/17539)
```

Comment:
The current CSV importing has the distributed implementation disabled by default. It may make sense to leave the last part of that sentence out.

Reply:
Will it be enabled for 1.1? Is a setting available for people to opt-in? Aside from "in a distributed fashion", is the rest of this accurate?

Reply:
We don't know if it will be enabled for 1.1. That will depend on the testing we will do to see if/how it scales to larger clusters. Currently it's panic'ing with sf-100 on 4 nodes. It is available for opt in as an option in the sql statement. Yes, the rest is accurate.

Reply:
Done.